go.mod

```diff
@@ -13,13 +13,13 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.3.0
 	github.com/containers/buildah v1.31.1-0.20230710135949-9c9a344b9874
-	github.com/containers/common v0.55.1-0.20230713173316-9e5d4a690901
+	github.com/containers/common v0.55.1-0.20230719211449-b94e030a5ea0
```

Comment:
This should really be v0.56.*, not v0.55.*.   The v0.55 c/common is for Podman v4.6, and is not meant for Podman upstream/v4.7-dev.

Reply:
I just used the latest hash from c/common's main branch. There isn't a v0.56 branch in c/common yet so I'm not sure what to do here.

Reply:
@TomSweeneyRedHat, that's go-mod slang which sometimes guesses the wrong version.  The only source of truth is the short ID after the last dash which points to the specific commit.